### PR TITLE
Fixes #12884 - Allow importing subnets from smart proxy

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/models/subnet.rb [hunk 1]
```diff
@@ -193,7 +193,7 @@ def self.import(proxy)
 
       next if exists?(attrs)
       attrs.merge!(parse_dhcp_options(s['options'])) if s['options'].present?
-      new(attrs.update(:dhcp => proxy))
```
Comment (dLobatog) [inline app/models/subnet.rb:196]: Why not just adding `dhcp', 'dns' and`tftp` to attr_accessible? I think that'd be the only change required.

**Comment by tbrisker** [comment]
@dLobatog Changed this to add the missing attributes as you suggested.

**Comment by dLobatog** [comment]
:+1: to merge when Jenkins comes back

**Comment by dLobatog** [comment]
[test]

**Comment by dLobatog** [comment]
Merged as e42c1105d8ea4266f12a1bd076d405af00e09ea7, thanks @tbrisker!

**Pull request closed without merging**